### PR TITLE
Export LIB_DIR

### PR DESCRIPTION
**Pull request by sophia-guo**
Fix https://github.com/adoptium/aqa-tests/issues/4914

### CODE REVIEW

**Comment by sophia-guo** [comment]
I think this should work in the github runner environment, but hard to test unless it's merged.

File: makefile [hunk 1]
```diff
@@ -41,8 +41,7 @@ endif
 ifeq ($(CYGWIN),1)
 	LIB_DIR:=$(shell cygpath -w $(LIB_DIR))
 endif
-LIB_DIR:=$(subst \,/,$(LIB_DIR))
-
+export LIB_DIR:=$(subst \,/,$(LIB_DIR))
```
Comment (llxia) [inline makefile:44]: It is not recommended to change values and export in one cmd. Please separate the code into two lines.

**Comment by llxia** [comment]
Could you test this change locally and provide console output? Thanks

File: makefile (hunk 1 above)
Comment (sophia-guo) [reply makefile:44]: I'm just trying to keep the same style https://github.com/adoptium/TKG/blob/4c1094f7e8fdab2794a712c819beec812dc43757/makefile#L22

**Comment by sophia-guo** [comment]
@LongyuZhang mentioned https://github.com/adoptium/aqa-tests/issues/4914 is also the cause of external tests ( using docker image) failures hyc-runtimes-grinder 36901, this PR combined the former one fix the issue grinder  hyc-runtimes-grinder  36903. The external functional tests is the environments similar to locals without jenkins environment set up.

**Comment by LongyuZhang** [comment]
More details about the above mentioned external tests, inside docker, we run functional tests similar to local test, hyc-grinder 36901 failed since no lib_dir, while hyc-grinder 36903 passed with this PR.

**Comment by sophia-guo** [comment]
@llxia Grinders to avoid possible pipeline broken
linux: https://ci.adoptium.net/view/Test_grinder/job/Grinder/8218/
linux parallel: https://ci.adoptium.net/view/Test_grinder/job/Grinder/8219/
windows https://ci.adoptium.net/view/Test_grinder/job/Grinder/8220/
windows parallel https://ci.adoptium.net/view/Test_grinder/job/Grinder/8221/

**Review by llxia — APPROVED** [review]

**Pull request merged**